sdk/tests/tutorials/properties/test_transaction_properties.py

```diff
@@ -0,0 +1,135 @@
+            code=self.code,
+            display_name=self.code,
+            base_currency="GBP",
+            created=effective_date,
+        )
+
+        # create portfolio
+        try:
+            self.transaction_portfolios_api.create_portfolio(
+                scope=self.scope,
+                create_transaction_portfolio_request=create_portfolio_request,
+            )
+        except lusid.ApiException as e:
+            if json.loads(e.body)["name"] == "PortfolioWithIdAlreadyExists":
+                logging.info(
+                    f"Portfolio {create_portfolio_request.code} already exists"
+                )
+
+    def create_txn_with_property(self, instrument_id, property_value):
+        # setup the transaction
+        effective_date = datetime(2020, 12, 1, 0, 0, tzinfo=pytz.utc)
+        txn = models.TransactionRequest(
+            transaction_id="TXN001",
+            type="Buy",
+            instrument_identifiers={"Instrument/default/Figi": instrument_id},
+            transaction_date=effective_date,
+            settlement_date=effective_date,
+            units=1000,
+            transaction_price=models.TransactionPrice(price=100, type="Price"),
+            total_consideration=models.CurrencyAndAmount(amount=1, currency="GBP"),
+            exchange_rate=1,
+            transaction_currency="GBP",
+            properties={
+                f"Transaction/{self.scope}/{self.code}": lusid.PerpetualProperty(
+                    key=f"Transaction/{self.scope}/{self.code}",
+                    value=lusid.PropertyValue(label_value=property_value),
+                )
+            },
+        )
+
+        return self.transaction_portfolios_api.upsert_transactions(
+            scope=self.scope, code=self.code, transaction_request=[txn]
+        )
+
+    def get_transaction(self, scope, code):
+        return self.transaction_portfolios_api.get_transactions(scope=scope, code=code)
+
+    def test_transaction_property(self):
+        # Value for our property
+        transaction_tax_details = '{"Tax": 1.0, "Rate": 0.01, "Schedule": "A"}'
+
+        # Setup property and portfolio
+        self.create_transaction_property()
+        self.create_portfolio()
+
+        # Setup transaction with txn tax details as the property value
+        response = self.create_txn_with_property(
+            "BBG000FD8G46", transaction_tax_details
+        )
+        self.assertIsNotNone(response)
+
+        # Get transaction with property
+        txn_response = self.get_transaction(scope=self.scope, code=self.code)
+        self.assertIsNotNone(txn_response)
+
+        # Parse property value from txn and assert is equal to original object
+        queried_property_value = (
+            txn_response.values[0]
+            .properties[f"Transaction/{self.scope}/{self.code}"]
+            .value.label_value
+        )
+        self.assertIsNotNone(queried_property_value)
```

Comment:
Could we demo how we can extract the individual fields from the JSON ?

Reply:
That's been added to the example now. It reads a dictionary in this case, which is then dumped into a string representation that is used as the property value and then parsed back using json.loads().